StRoot/StPicoEvent/StPicoEvent.h

```diff
@@ -154,7 +154,31 @@ class StPicoEvent : public TObject {
   Int_t    nTofT0() const                { return (Int_t)mNTofT0; }
   /// Return z position of the primary vertex estimated by VPD
   Float_t  vzVpd() const                 { return mVzVpd; }
-
+  // In the following, the factors of 1000. serve to convert times back and forth between external nsec and internal psec
+  /// Return number of particles before T0 outlier rejection
+  Int_t    nTofT0Can() const		 { return (Int_t)mNTofT0Can; }
+  /// Return event start time
```

Comment:
Mark units please
```suggestion
  /// Return event start time in nano seconds
```
and below